pages/index.tsx

```diff
@@ -1,12 +1,16 @@
 import MapControl from '@components/MapControl';
 import MapExtraMethodsProvider from '@context/map';
 import { clearSearch } from '@plugins/store/slices/search-query';
+import { useWallet } from '@solana/wallet-adapter-react';
 import { useEffect } from 'react';
 import { MapProvider } from 'react-map-gl';
 import { useDispatch } from 'react-redux';
 
 export default function Home() {
   const dispatch = useDispatch();
+  const walletInfo = useWallet();
```

Comment:
I think `walletInfo` and `console.log` can be removed. I forgot to take them off after I pushed my stuff coz I was testing some stuff out.